ethereum/evmtool/src/main/java/org/hyperledger/besu/evmtool/EvmToolCommand.java

```diff
@@ -143,10 +149,27 @@ void setBytes(final String optionValue) {
   final Boolean showJsonAlloc = false;
 
   @Option(
-      names = {"--nomemory"},
-      description = "Disable showing the full memory output for each op.",
-      scope = INHERIT)
-  final Boolean noMemory = false;
+      names = {"--memory", "--trace.memory"},
```

Comment:
From the PR conversation, sounds like this should still be --nomemory? Or is it the comment that's wrong? https://github.com/hyperledger/besu/pull/5030/#issuecomment-1435324768

Reply:
Line 156 added the "negatable" flag to this option and I added the proper regexp support at line 219.  This boolean flag is now `--nomemory` or ``--memory`, or `--trace.nomemory` or `--trace.memory` and the boolen is set based on the particular flag.

Reply:
ah ok, that makes sense now thanks.